save.go

```diff
@@ -52,6 +52,8 @@ vendor experiment.
 
 If -v is given, verbose output is enabled.
 
+If -m is given, only save out to the minor go version to Godeps.json.
```

Comment:
s/minor/major/

For example, Go 1.5 is a major release, and 1.5.1 is a minor revision.
See https://golang.org/doc/devel/release.html.